apps/emqx_machine/src/emqx_machine.erl

```diff
@@ -67,3 +73,12 @@ start_sysmon() ->
             _ = application:ensure_all_started(system_monitor, temporary),
             ok
     end.
+
+node_status() ->
+    emqx_json:encode(#{ backend => mria_rlog:backend()
+                      , role    => mria_rlog:role()
+                      }).
+
+update_vips() ->
+    VIPs = [mnesia_tm, mnesia_locker | mria_status:shards_up()],
```

Comment:
Any other processes to add here?

Reply:
Could this be made dynamic at runtime?
E.g. in some gen_server init callback it registers self

Reply:
It is dynamic, it will run every 10 seconds as specified here: https://github.com/emqx/emqx/pull/6923/files#diff-38e3ba807429931ca73c2ccabc0a9311fc6e2e63bc29bec685e79bff95a3d63eR65

Reply:
Done. Now it can be done using `system_monitor:add_vip/1` function.